charts/policy-controller/templates/webhook/deployment_webhook.yaml

```diff
@@ -12,6 +12,13 @@ spec:
     matchLabels:
       {{- include "policy-controller.selectorLabels" . | nindent 6 }}
       control-plane: {{ template "policy-controller.fullname" . }}-webhook
+
+{{- if .Values.deployment.strategy }}
+  strategy:
+{{ toYaml .Values.deployment.strategy | trim | indent 4 }}
+    {{ if eq .Values.deployment.strategy.type "Recreate" }}rollingUpdate: null{{ end }}
```

Comment:
let's default for the default value when you not set this field, and drop the `{{- if .Values.deployment.strategy }}` i dont think we need to add any extra complexity for this

Reply:
Okay - think I've done that although my templating knowledge is not amazing!